server/src/main/java/org/apache/iotdb/db/mpp/sql/planner/plan/node/metedata/read/SchemaMergeNode.java

```diff
@@ -18,19 +18,16 @@
  */
 package org.apache.iotdb.db.mpp.sql.planner.plan.node.metedata.read;
 
-import org.apache.iotdb.db.mpp.common.header.ColumnHeader;
 import org.apache.iotdb.db.mpp.sql.planner.plan.node.PlanNode;
 import org.apache.iotdb.db.mpp.sql.planner.plan.node.PlanNodeId;
 import org.apache.iotdb.db.mpp.sql.planner.plan.node.PlanNodeType;
 import org.apache.iotdb.db.mpp.sql.planner.plan.node.PlanVisitor;
-import org.apache.iotdb.db.mpp.sql.planner.plan.node.process.ProcessNode;
-import org.apache.iotdb.tsfile.file.metadata.enums.TSDataType;
 
 import java.nio.ByteBuffer;
 import java.util.ArrayList;
 import java.util.List;
 
-public class SchemaMergeNode extends ProcessNode {
+public class SchemaMergeNode extends AbstractSchemaMergeNode {
```

Comment:
A kindly suggestion, maybe we can give it a more concrete name like `SeriesSchemaMergeNode`? Because we already have a extender of `AbstractSchemaMergeNode` whose name is `CountSchemaMergeNode`.

Reply:
Yeah, it's a comfortable name, I will fix it.